Misc/NEWS

```diff
@@ -9,6 +9,7 @@ What's New in Python 3.7.0 alpha 1?
 
 Core and Builtins
 -----------------
+- bpo-29649: struct.pack_into check boundary error message didn't respect offset.
```

Comment:
This needs to be moved to "Library" section. And please add an empty line after the section header.